runtime/interpreter/value.go

```diff
@@ -15310,10 +15478,10 @@ func (v *DictionaryValue) ForEachKey(
 	keyType := v.SemaType(interpreter).KeyType
 
 	iterationInvocation := func(key Value) Invocation {
-		var self MemberAccessibleValue = v
 		return NewInvocation(
 			interpreter,
-			&self,
+			nil,
+			nil,
```

Comment:
This looks like a "bug-fix", i.e. is the reason for the change that the key-iteration callback does not need to have the `self` value declared?

Reply:
Yes, the key iteration function should not have a `self` value declared. I don't think this was abusable because use of `self` would have failed at checking time but we were allocating a variable unnecessarily here.